api/client/utils_test.go

```diff
@@ -50,6 +50,7 @@ func setupServer(t *testing.T) func() {
 		GrpcWebProxyAddress: grpcWebProxyAddress,
 		RepoPath:            repoPath,
 		GatewayHostAddr:     gatewayHostAddr,
+		MaxMindDBFolder:     "../../iplocation/maxmind",
```

Comment:
This new `MaxMindDBFolder` is mostly for integration testing purposes. The default value (empty string) is the correct one for the baked DockerHub image, so no need to tune anything.